hack/tests/scaffolding/scaffold-memcached.go

```diff
@@ -271,6 +272,15 @@ func insertGoModReplace(repo, path, sha string) ([]byte, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to read go.mod")
 	}
+	// Remove all replace lines in go.mod.
+	replaceRe := regexp.MustCompile(fmt.Sprintf("(replace )?%s =>.+", repo))
+	idxs := replaceRe.FindAllIndex(modBytes, -1)
+	lenDiff := 0
+	for _, idx := range idxs {
+		modBytes = append(modBytes[:idx[0]-lenDiff], modBytes[idx[1]-lenDiff:]...)
```

Comment:
Since `slice[index:]` includes the character at `index`, should the second part be `idx[1]-lendiff` or `idx[1]-lendiff+1`?

Reply:
Not exactly sure what gave me the idea that doing `re.ReplaceAll()` manually was better than just calling `re.ReplaceAll()`...

To answer your question, the former: `modBytes[idx[1]-lenDiff]` is the first byte following the matched bytes, so `modBytes[idx[1]-lenDiff:]` contains everything following the match.